src/Build.cmd

```diff
@@ -51,17 +72,20 @@ SET OutDir=%CMDHOME%\..\Binaries\%CONFIGURATION%
 
 set STEP=VSIX
 
-if "%BuildOrleansNuGet%" == "false" (
+if NOT "%BuildOrleansVsix%" == "true" (
```

Comment:
I was talking to @sergeybykov, and we think the expectation is that the VSIX should be built by default if you just run the script after downloading the code. We do prefer that instead of failing the build if this cannot be built due to dependencies, that we display a warning instead. Nevertheless failing the build should be preferable to having to know that you need to opt into it with an environment variable.
Do you think it's easy to test whether the dependencies are installed and display a warning? Otherwise let's change the default and allow users to have to opt out instead.